packages/auth/src/access/types.ts

```diff
@@ -1,35 +1,27 @@
-export enum AccessType {
-  IncomingPayment = 'incoming-payment',
-  OutgoingPayment = 'outgoing-payment',
-  Quote = 'quote'
-}
-
-export enum Action {
-  Create = 'create',
-  Read = 'read',
-  ReadAll = 'read-all',
-  List = 'list',
-  ListAll = 'list-all',
-  Complete = 'complete'
-}
+import {
+  AccessType,
+  AccessTypeMapping,
+  Action,
+  ActionMapping
+} from 'open-payments/dist/types'
 
 interface BaseAccessRequest {
   actions: Action[]
   identifier?: string
 }
 
 export interface IncomingPaymentRequest extends BaseAccessRequest {
-  type: AccessType.IncomingPayment
+  type: 'incoming-payment'
   limits?: never
 }
 
-interface OutgoingPaymentRequest extends BaseAccessRequest {
-  type: AccessType.OutgoingPayment
+export interface OutgoingPaymentRequest extends BaseAccessRequest {
+  type: 'outgoing-payment'
   limits?: OutgoingPaymentLimit
 }
 
-interface QuoteRequest extends BaseAccessRequest {
-  type: AccessType.Quote
+export interface QuoteRequest extends BaseAccessRequest {
```

Comment:
It might be worth auditing what's actually used in this file.
I think some of these types and `is*` check functions could be removed

Reply:
Removed the is* methods that are not used